cdap-ui/app/cdap/components/AbstractWidget/SchemaEditor/index.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/styles/withStyles';
+import ThemeWrapper from 'components/ThemeWrapper';
+import {
+  SchemaManager,
+  INode,
+  ISchemaManager,
+  IOnChangeReturnType,
+} from 'components/AbstractWidget/SchemaEditor/Context/SchemaManager';
+import { ISchemaType } from 'components/AbstractWidget/SchemaEditor/SchemaTypes';
+import {
+  IFlattenRowType,
+  IFieldIdentifier,
+  IOnChangePayload,
+} from 'components/AbstractWidget/SchemaEditor/EditorTypes';
+import { FieldsList } from 'components/AbstractWidget/SchemaEditor/FieldsList';
+import {
+  SchemaValidatorConsumer,
+  SchemaValidatorProvider,
+} from 'components/AbstractWidget/SchemaEditor/SchemaValidator';
+import { dumbestClone as cloneDeep } from 'services/helpers';
```

Comment:
can we use this just as dumbestClone and not alias it with cloneDeep? Might be easier to detect if we need to refactor later simply through search

Reply:
When renaming the function for some reason typescript created the alias here instead of the helpers. Have changed.